client/controllers.js

```diff
@@ -1,4 +1,43 @@
-var micControllers = angular.module('micControllers', []);
+var micControllers = angular.module('micControllers', ['ribbitBaseRTC', 'ngSanitize']);
+
+// Audience RTC:
+//  - Acquires the user's local media stream right away, upon instantiating
+//  - Connects specifically to the room's presenter.
+// NOTE: The connect method must be passed a room that has a presenter property.
+micControllers.factory('audienceRTC', function (baseRTC) {
```

Comment:
Why do we have this code here and in client/audience-rtc.js? Which copy do we actually use?